jest.config.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
     },
     moduleFileExtensions: ['js', 'jsx', 'ts', 'tsx', 'json'],
     moduleDirectories: ['node_modules', 'app/node_modules'],
-    setupFiles: ['./internals/scripts/CheckBuildsExist.js'],
+    setupFiles: [
+        './internals/scripts/CheckBuildsExist.js',
+        './test/loadModules.ts',
```

Comment:
Why is this needed?

Reply:
Because otherwise we get a ReferenceError on TextDecoder on test files that import the cbor dependency.